src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/AuthTest.kt

```diff
@@ -0,0 +1,69 @@
+package uk.gov.justice.digital.hmpps.approvedpremisesapi.integration
+
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.util.JwtProvider
+
+class AuthTest : IntegrationTestBase() {
```

Comment:
Not sure if you have a future PR but we would typically also protect endpoints with a role; would should check the token contains the correct role (aka authority) and return a 403 if it doesn't.

Reply:
For reference spring boot - [Method Secuity](https://docs.spring.io/spring-security/reference/servlet/authorization/method-security.html)

Reply:
This is still a TBD for us yet (which authority to use, potentially we'll need a new one) so will add once we've come to a decision